amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1864,11 +1864,12 @@ VX_API_ENTRY vx_node VX_API_CALL vxExtrppNode_Brightness(vx_graph graph, vx_tens
             (vx_reference)pDst,
             (vx_reference)alpha,
             (vx_reference)beta,
-            (vx_reference)layout,
+            (vx_reference)inputLayout,
+            (vx_reference)outputLayout,
             (vx_reference)roiType,
             (vx_reference)NBATCHSIZE,
```

Comment:
please don't use capitals for parameters

Reply:
We have used capital for the BATCHSIZE and DEVTYPE variables alone because the vx_scalar is created in this function